drivers/adc/adc_stm32.c

```diff
@@ -642,7 +687,7 @@ static int adc_stm32_check_acq_time(uint16_t acq_time)
 		return 0;
 	}
 
-	LOG_ERR("Conversion time not supportted.");
+	LOG_ERR("Conversion time not supported.");
```

Comment:
You might want to split this into its own commit also.

Reply:
@mbolivar-nordic I can do that, would you recommend to just remove this and the other changes and force update the single comment. Then create new comments for the other changes, like the fix of this typo in this case?

Reply:
How people split up commits while rebasing is a personal decision, so I can't really offer you a "recommendation" in that sense. Beyond individual preference, tooling varies: for example, I use [magit](https://magit.vc/) instead of the command line.

However, one simple way to handle this since the PR contains a single commit is to put all its changes back into the working tree:

```
git checkout feature/stm32f407-adc
git reset HEAD~
```

Then you can split changes in a single file into their own commits using `git add -p`, e.g. for this particular comment:

```
git add -p drivers/adc/adc_stm32.c
< follow instructions to just stage this hunk>
git commit
<write a commit comment describing the typo fix>
```

Then keep adding additional commits for the other changes you want to make.

But again this sort of thing is subject to taste, and I would do it in a different way using my own favorite tools.